File: fixtures/rust_output/post_with_data_raw.rs
extern crate reqwest;

fn main() -> Result<(), Box<dyn std::error::Error>> {
    let res = reqwest::Client::new()
        .post("http://example.com/post")
        .body("msg1=wow&msg2=such&msg3=@rawmsg")
        .send()?
        .text()?;
    println!("{}", res);

    Ok(())
}

File: test/fixtures/cfml/post_with_data_raw.js
httpService = new http();
httpService.setUrl("http://example.com/post");
httpService.setCharset("utf-8");
httpService.setMethod("POST");
httpService.addParam(type="body", value="msg1=wow&msg2=such&msg3=@rawmsg");
result = httpService.send().getPrefix();
writeDump(result);
